src/main/resources/templates/error.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+<head> </head>
+<body>
+    Wrong page. This page does not exist on our website
+    To go back home click <a href="/">here</a>
```

Comment:
Added a link back to the home page so users can go back from the error page to where they began